pkg/provisioner/ironic/raid/raid.go

```diff
@@ -0,0 +1,161 @@
+package raid
```

Comment:
I like that this is in a separate file from the provisioner, but I wonder if it needs to be a separate package like this. Could we just move raid.go up one directory and make it part of the ironic provisioner package? That would let us keep most of the functions and types private. Unless you think something else may want to import these types?

Reply:
Done.